sys/net/link_layer/ieee802154/Kconfig

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2019 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+menuconfig KCONFIG_MODULE_IEEE802154
+    bool "IEEE802.15.4 configurations"
+    depends on MODULE_IEEE802154
+    help
+        Configure IEEE802.15.4 module using Kconfig
+
+if KCONFIG_MODULE_IEEE802154
+
+    config IEEE802154_DEFAULT_SUBGHZ_CHANNEL
+        int "IEEE802.15.4 default sub-GHZ channel"
+        default 5
+        range 0 11
```

Comment:
This depends on the modulation and the frequency band. (see e.g. IEEE 802.15.4g-2012 Table 68d).
It's a mess - I'd just give up and drop the `range`.

Reply:
done!